packages/presentation/src/file.ts

```diff
@@ -284,3 +284,13 @@ async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: str
     })
   }
 }
+
+export async function getJsonOrEmpty (file: string, name: string): Promise<any> {
+  try {
+    const fileUrl = getFileUrl(file, name)
+    const resp = await fetch(fileUrl)
+    return await resp.json()
+  } catch {
+    return []
```

Comment:
nit: for json expected default value would be an empty object. if you need an empty array, consider adding a `default` attribute that would allow to control which value to return

Reply:
oops! Fixed, thanks.